frontend/src/components/shared/Button/styled-components.ts

```diff
@@ -84,7 +85,7 @@ export const StyledBaseButton = styled.button<RequiredButtonProps>(
     margin: 0,
     lineHeight: theme.lineHeights.base,
     color: "inherit",
-    width: fluidWidth ? "100%" : "auto",
+    width: useContainerWidth || fluidWidth ? "100%" : "auto",
```

Comment:
If `useContainerWidth` and `fluidWidth` do the same, why do we keep both? Also, `fluidWidth` seems not to be mentioned in [docs](https://docs.streamlit.io/library/api-reference/widgets/st.button)

Reply:
Seems like someone has done an API to make `Button` stretch to `width: 100%`, I've used fluid width instead, but kept `useContainerWidth` on backend as is required by the spec 👍